src/main/java/MyParser/Parser.java

```diff
@@ -103,12 +103,12 @@ private boolean isIn(String ch, String set) {
      * private boolean isVariable(String val) { String ch; for(int i=0;i<val.length();i++) {
      * ch=val.substring(i,i+1); if (isIn(ch,ALPHA)==false) return false; } return true; }
      */
-
+    // TODO NumberUtils anyone?
```

Comment:
Rather no NumberUtils but an api that is enough strongly typed to not check.

Reply:
Something in that direction. I saw this code 5 times in a row. Where something is checked to be a number.